### PR TITLE
Elasticsearch example for DC/OS 1.10

### PR DESCRIPTION
**Pull request by ondrej-smola**
Admin router link does not work (same as for 1.9)

### CODE REVIEW

File: elasticsearch/1.10/README.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+By Deploying, you agree to the Terms and Conditions https://mesosphere.com/catalog-terms-conditions/#community-services
+This DC/OS Service is no longer maintained. See beta-elastic. The ElasticSearch DCOS Service implementation is alpha and there may be bugs, incomplete features, incorrect documentation or other discrepancies.
+Continue installing? [yes/no] yes
+Installing Marathon app for package [elasticsearch] version [1.0.1-2]
+Elasticsearch is staging. Check Marathon and the ES management front-end for status.
+```
+
+After this, you should see the Elasticsearch service running via the `Services` tab of the DC/OS UI:
+
+![Elasticsearch DC/OS service](img/services.png)
```
Comment (mattj-io) [inline elasticsearch/1.10/README.md]: I think this is the 1.9 GUI, as this layout has changed in 1.10 - can you update the image with an image of the latest GUI ?

**Comment by ondrej-smola** [comment]
My bad, i am maintaining 1.9 clusters using ssh tunnel to localhost and it cached UI components.

**Comment by ondrej-smola** [comment]
Also found there packages elastic, elastic-beta and elasticsearch ... i was testing elasticsearch but it seems like elastic is the new version?

**Comment by mattj-io** [comment]
elastic-beta is the new framework using the SDK - at some point we will update examples for those too, but this package is fine for the minute.

**Pull request merged**